code/modules/clothing/gloves/color.dm

```diff
@@ -8,6 +8,11 @@
 	item_color="yellow"
 	burn_state = -1 //Won't burn in fires
 
+/obj/item/clothing/gloves/color/yellow/syndicate //Actually red, this is what you get when you just arbitrarily decide that one color is better than the rest!

```

Comment:
setting the siemens_coefficient and burn_state on red gloves would actually be less lines, NERD.